EasyClangComplete.py

```diff
@@ -78,6 +78,21 @@ def run(self, edit):
             log.debug(" Nothing to clean")
 
 
+class ApplyFixitCommand(sublime_plugin.TextCommand):
+    """Command to apply fixit at cursor position."""
+
+    def run(self, edit):
+        """All the actual processing is done in on_text_command."""
```

Comment:
Why not doing it here? It seems to complicate the flow. There is nothing that you are missing here that you are using in the `on_text_command` or am I missing anything?

This will allow moving the code out of the main file eventually as it grows and I don't like it (that is my fault, I should have kept it to the minimum from the start)